lib/countly.js

```diff
@@ -1342,6 +1342,32 @@
         if(!Countly.ignore_visitor && !hasPulse)
             heartBeat();
     };
+
+    /**
+    * Show specific widget popup by the widget id
+    * @param {string} id - id value of related feedback widget, you can get this value by click "Copy ID" button in row menu at "Feedback widgets" screen
+    */
+    Countly.show_feedback_popup = function(id) {
+        // define xhr object
+        var getWidget = new XMLHttpRequest();
```

Comment:
instead of creating another function to send http request, maybe we can modify existing one:
https://github.com/Countly/countly-sdk-web/blob/master/lib/countly.js#L1715

To accept url as param (or fallback to current default one), and return additional third value - response text in the callback.

Reusing functions would enable us to changing code in one place when we want to change things, as well as allow us to keep library much smaller.